src/pages/Cart/index.js

```diff
@@ -1,17 +1,62 @@
 import React from 'react'
-import { H1 } from '../../components/Typography'
+import { connect } from 'react-redux'
+
 import Layout from '../../components/Layout'
+import { H1 } from '../../components/Typography'
+import CartItemComponent from './components/cartItem'
+import { CartItemsWrapper } from './styled'
+import { removeProduct } from '../../store/cartItems/actions'
+
+class CartView extends React.Component {
+  handleRemoveFromCart = (productId, evt) => {
+    evt.preventDefault()
```

Comment:
event can be prevented directly in handler of src/pages/ProductList/components/Product/index.js so this fn can have just one argument (or this.props.removeProduct can be passed as CartItemComponent's onRemoveFromCart prop directly, so this CartView class can become stateless functional component)

Reply:
You mean `src/pages/Cart/components/cartItem/index.js` instead of file, which you mentioned `src/pages/ProductList/components/Product/index.js`, right? When i am using `mapDispatchToProps`, do i have use `mapStateToProps`? 

I am trying to move `handleRemoveFromCart` to cartItem component, but actually there became a lot of errors in console :(